records_mover/records/targets/data_url.py

```diff
@@ -22,14 +22,18 @@ def __init__(self,
                  output_loc: BaseFileUrl,
                  records_format: Optional[BaseRecordsFormat]) -> None:
         if records_format is None:
```

Comment:
Since if any hint was provided in, a RecordsFormat object was created, we would have skipped the inference entirely.